ktx-library-codelab/build.gradle

```diff
@@ -45,23 +43,23 @@ buildscript {
     }
     repositories {
         google()
-        jcenter()
+        mavenCentral()
         maven {
-            url "http://dl.bintray.com/kotlin/kotlin-eap"
+            url "https://dl.bintray.com/kotlin/kotlin-eap"
```

Comment:
can you check if it will work without this? It might not be needed here and below

Reply:
Hi Wojtek, the build failed with an error message regarding unsecured protocol.

Reply:
What I meant was to remove that entry completely

Reply:
Removed.